modules/auxiliary/admin/http/atlassian_confluence_auth_bypass.rb

```diff
@@ -0,0 +1,138 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Atlassian Confluence Data Center and Server Authentication Bypass via Broken Access Control',
+        'Description' => %q{
+          This module exploits a broken access control vulnerability in Atlassian Confluence servers leading to an authentication bypass.
+          A specially crafted request can be create new admin account without authentication on the target Atlassian server.
+        },
+        'Author' => [
+          'Unknown', # exploited in the wild
+          'Emir Polat' # metasploit module
+        ],
+        'References' => [
+          ['CVE', '2023-22515'],
+          ['URL', 'https://confluence.atlassian.com/security/cve-2023-22515-privilege-escalation-vulnerability-in-confluence-data-center-and-server-1295682276.html'],
+          ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2023-22515'],
+          ['URL', 'https://attackerkb.com/topics/Q5f0ItSzw5/cve-2023-22515/rapid7-analysis']
```

Comment:
['URL', 'https://twitter.com/Dinosn/status/1712158087862218958']

Reply:
This PoC is not enough for explotation.